src/napari_micromanager/main_window.py

```diff
@@ -329,12 +335,16 @@ def _add_explorer_positions_layers(
             layer = self.viewer.add_image(z, name=f"{fname}_{id_}", blending="additive")
             layer.visible = False
 
+            # set layer scale
+            layer.scale = self._get_scale_from_sequence(sequence, layer)
+
             # add metadata to layer
             layer.metadata["mode"] = meta.mode
             layer.metadata["useq_sequence"] = sequence
             layer.metadata["uid"] = sequence.uid
             layer.metadata["grid"] = pos.split("_")[-3]
             layer.metadata["grid_pos"] = pos.split("_")[-2]
+            layer.metadata["scale"] = layer.scale
```

Comment:
is this "just in case" or do you use it elsewhere?  adding `layer.scale` to `layer.metadata['scale']` feels like an unnecessary duplication? (we already have it at layer.scale, no?)

Reply:
no you're right, is a duplication.